class.photon.php

```diff
@@ -695,6 +700,12 @@ public function filter_srcset_array( $sources, $size_array, $image_src, $image_m
 			$newsources = null;
 
 			foreach ( $multipliers as $multiplier ) {
+
+				/** This filter is already documented in class.photon.php */
+				if ( apply_filters( 'jetpack_photon_skip_image', false, $url ) ) {
```

Comment:
I wonder if this would be better up near line 668/673? If the URL is set to skip Photon, then skip out at the `if (is_array($multipliers)` line? If an image is excluded, wouldn't we expect all multiples (all are Photon-based) would not be used either?

Thoughts?